### PR TITLE
Update sources & add shipping address

### PR DESCRIPTION
**Pull request by michael-giorgaras-cko**

### CODE REVIEW

File: spec/components/schemas/Sessions/NetworkTokenSource.yaml [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+        example: 2025
+        minLength: 4
+        maxLength: 4
+      token_type:
+        type: string
+        description: The type of token
+        enum:
+          - vts
+          - mdes
+          - applepay
```
Comment (helder-goncalves-cko) [inline spec/components/schemas/Sessions/NetworkTokenSource.yaml]: Need to check if `applepay` and `googlepay` are valid for 3DS 2.0

File: spec/components/schemas/Sessions/NetworkTokenSource.yaml [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+          - googlepay
+      cryptogram:
```
Comment (helder-goncalves-cko) [inline spec/components/schemas/Sessions/NetworkTokenSource.yaml]: `cryptogram` is not valid here

File: spec/components/schemas/Sessions/NetworkTokenSource.yaml (hunk 1 above)
Comment (michael-giorgaras-cko) [reply spec/components/schemas/Sessions/NetworkTokenSource.yaml]: Removed them to align with NT I guess

File: spec/components/schemas/Sessions/NetworkTokenSource.yaml (hunk 2 above)
Comment (michael-giorgaras-cko) [reply spec/components/schemas/Sessions/NetworkTokenSource.yaml]: Removed it

**Pull request merged**